internal/filetokensource/config.go

```diff
@@ -9,10 +9,6 @@ var ErrTokenPathRequired = errors.New("file token source: TokenPath required")
 type Config struct {
 	// TokenPath is the path to the source jwt token.
 	TokenPath string
-
-	// NoReuseToken if enabled disables reusing of tokens while they're still valid.
```

Comment:
i was asking if we had this lol it seems we did?

Reply:
This was only for the file token source. There wasn't an equivalent for the exchanged token. So this wouldn't have done what you wanted.